### ISSUE TITLE
threddsDev pqacts need to be cleaned up

### ISSUE THREAD

**Issue by lesserwhirls** [Unidata/TdsConfig#36]
The pqacts for threddsDev need some attention, as I think they might be out of sync with what is in the repo. Specifically, not all pqacts maintained in the repo are actually used in ldmd.conf:

~~~bash
# THREDDS processing
EXEC    "pqact -f NGRID|CONDUIT|HRS|FNMOC etc/TDS/pqact.forecastModels"
EXEC    "pqact -f NGRID|CONDUIT etc/TDS/pqact.forecastProdsAndAna"
EXEC    "pqact -f HRS|NIMAGE|NOTHER etc/TDS/pqact.satellite"
EXEC    "pqact -f HRS|FNEXRAD|NNEXRAD|CRAFT etc/TDS/pqact.radars"
EXEC    "pqact -f HRS|IDS|DDPLUS etc/TDS/pqact.obsData"
EXEC    "pqact -f NGRID|FSL2 etc/TDS/pqact.modelsHrrr"
EXEC    "pqact -f WMO|HRS|NGRID|CONDUIT|EXP etc/TDS/pqact.testDatasets"
EXEC    "pqact -f IDS|HRS|NOTHER etc/TDS/pqact.ryan-test"
~~~

None of the `pqact.forecastModels-*` pqacts are used by the ldm. My guess is that we can just remove them, as I think they were used for testing at some point (by me). Any objections if they go away?

**Comment by dopplershift** [comment]
I think those forecastModels-* were used to try to deal with some processing backlog that no longer exists on the new lead.

**Comment by dopplershift** [comment]
I say blow em away.

**Comment by lesserwhirls** [comment]
ok, [PR](https://github.com/Unidata/TdsConfig/pull/37) submitted with some additional things. What about `pqact.ryan-test`? That one isn't checked into github.

**Comment by dopplershift** [comment]
Right, that's the one I hack on live on the machine...though I'd be sad if it got deleted...

Can you add?

**Comment by lesserwhirls** [comment]
Yep - added it just now.

**Comment by dopplershift** [comment]
Closed by #37 .
